evalai/dramatiq.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from dramatiq.middleware import AgeLimit, TimeLimit, Callbacks, Pipelines, Prometheus, Retries
+from dramatiq_sqs import SQSBroker
+
+
+broker = SQSBroker(
+    namespace="dramatiq_sqs_tests",
+    middleware=[
+        Prometheus(),
+        AgeLimit(),
+        TimeLimit(),
+        Callbacks(),
+        Pipelines(),
+        Retries(min_backoff=1000, max_backoff=900000, max_retries=96),
+    ],
+    endpoint_url=os.environ.get("AWS_SQS_ENDPOINT_URL", "http://localhost:9324"),
```

Comment:
change to "sqs:9324"

Reply:
@yashdusing The issue here is that we define sqs://9324 in docker.env file for dev env, so the locak dev env would work but Travis uses ElasticMQ on localhost:9324 only. So if we change it to sqs:9324, then the local dev env would work but travis build would fail.

Reply:
What do you think about this?
```python
if settings.DEBUG:
    sqs_endpoint = os.environ.get("AWS_SQS_ENDPOINT_URL", "sqs:9324")
else:
    sqs_endpoint = os,environ.get("AWS_SQS_ENDPOINT_URL", "localhost:9324")
```

Reply:
yes, this sounds better. You may also add a comment to describe 'for travis', 'for local'